connect/api/src/main/java/org/apache/kafka/connect/connector/Connector.java

```diff
@@ -150,4 +152,15 @@ public Config validate(Map<String, String> connectorConfigs) {
      * @return The ConfigDef for this connector; may not be null.
      */
     public abstract ConfigDef config();
+
+    private void validateConfigDoesNotContainNull(Map<String, String> connectorConfigs) {
+        final String keysWithNullValue = connectorConfigs.entrySet().stream()
+                .filter(entry -> entry.getValue() == null)
+                .map(Map.Entry::getKey)
+                .collect(Collectors.joining(", "));
+
+        if (!keysWithNullValue.isEmpty()) {
+            throw new ConnectException(String.format("Null value found in config for key(s) %s", keysWithNullValue));
```

Comment:
Should this just be IllegalArgumentException?

Reply:
Thank you for the comment, updated it in the latest commit.